pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -15,24 +15,68 @@
 Contains the MPSPrep template.
 """
 
+import numpy as np
+
 import pennylane as qml
 from pennylane.operation import Operation
 from pennylane.wires import Wires
 
 
+def _mps_to_right_canonical_representation(mps, max_bond_dim):
+    """
+    Transform a MPS into a right-canonical MPS.
```

Comment:
What is a right-canonical MPS? Just curious.

Reply:
It is a property of the mps that guarantees that when the circuit matrix is created, it is unitary